docs/src/layout/page/demo/demo.jsx

```diff
@@ -1,7 +1,7 @@
 import {
     AppBar,
     Dropdown,
-    Header,
+    Typography,
```

Comment:
Are we not using this?  Only the import had to change, but no actual code? 😕 
If we're not using it, let's just remove it, yeah?

Reply:
We weren't using Header it looks like, but we are using Typography.

Reply:
The updated diff now shows `Typography` as already there, with some other removals only.  Whereas the diff I originally commented on showed  `Typography` being _added_, without adding corresponding usage in the component's `render()`, which, in my defense, seems suspicious.  I imagine this is likely a product of changing this file in various different 10.0.0 feature branches.

I'll go ahead and resolve this thread.  Nothing to see here ... move along.